bugwarrior/docs/services/github.rst

```diff
@@ -160,7 +160,7 @@ Comments are synchronized as annotations.
 To limit the amount of content synchronized into TaskWarrior (which can help to avoid issues with synchronization), use
 
  * ``annotation_comments=False`` (a global configuration) to disable synchronizing comments to annotations; and
- * ``github.body_length=0``` to disable synchronizing the Github Body UDA (or set it to a small value to limit size).
```

Comment:
I don't think we should suggest multiple ways to attain the same ends with no explanation of the benefits of making one choice over the other.

The advantage of `body_length` is that you might want to limit the length to a non-zero value. (For example to fix #304.)

So maybe something more like

    * either set ``github.body_length`` to limit size or include ``githubbody`` in ``static_fields`` in the ``[general]`` section to eliminate the UDA entirely

Reply:
To be clear: I'm suggesting that using static_fields is always a better solution than body_length=0. Perhaps we should even raise a warning to that effect in the validation function.

Reply:
Makes sense!